src/Mutator/Util/MutatorProfile.php

```diff
@@ -278,5 +284,9 @@
         'CastInt' => Mutator\Cast\CastInt::class,
         'CastObject' => Mutator\Cast\CastObject::class,
         'CastString' => Mutator\Cast\CastString::class,
+
+        // Unwrap
+        'UnwrapArrayMap' => Mutator\Unwrap\UnwrapArrayMap::class,
+        'UnwrapArrayFilter' => Mutator\Unwrap\UnwrapArrayFilter::class,
```

Comment:
@akondas @borNfreee @sanmai 

Is the order important here?

If not, maybe we can sort alphabetically?